liquibase-standard/src/main/java/liquibase/executor/jvm/JdbcExecutor.java

```diff
@@ -188,7 +188,27 @@ public void execute(final SqlStatement sql, final List<SqlVisitor> sqlVisitors)
     private void setParameters(final PreparedStatement pstmt, final RawParameterizedSqlStatement sql) throws SQLException {
         final List<Object> parameters = sql.getParameters();
         for (int i = 0; i < parameters.size(); i++) {
-            pstmt.setObject(i + 1, parameters.get(i));
+            Object parameter = parameters.get(i);
+            if(parameter instanceof ArrayList){
+                int finalI = i;
+                ((ArrayList<?>) parameter).forEach(param -> {
+                    try {
+                        if (param instanceof String) {
```

Comment:
The same logic is being repeated inside here and in the else.. could you create a private method to handle it and call it in both places?